<ui version="4.0" >
 <author>&lt;aumuell@reserv.at></author>
 <class>OrganizeCollectionDialogBase</class>
 <widget class="QWidget" name="OrganizeCollectionDialogBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>487</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="MinimumExpanding" hsizetype="MinimumExpanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Organize Files</string>
  </property>
  <widget class="QGroupBox" name="replacementGroup" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>250</y>
     <width>486</width>
     <height>151</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize" >
    <size>
     <width>0</width>
     <height>150</height>
    </size>
   </property>
   <property name="title" >
    <string>Character Replacement</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2" >
    <item>
     <widget class="QCheckBox" name="spaceCheck" >
      <property name="toolTip" >
       <string>If checked, convert spaces to underscores.</string>
      </property>
      <property name="text" >
       <string>&amp;Replace Spaces with Underscores</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="asciiCheck" >
      <property name="toolTip" >
       <string>If checked, replace characters that are unavailable in the 7-bit ASCII code.</string>
      </property>
      <property name="text" >
       <string>Restrict to &amp;ASCII</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="vfatCheck" >
      <property name="toolTip" >
       <string>If checked, replace characters that are incompatible with MS-DOS/VFAT file systems.</string>
      </property>
      <property name="text" >
       <string>VFAT Safe &amp;Names</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="textLabel1" >
        <property name="text" >
         <string>Replace</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KLineEdit" name="regexpEdit" >
        <property name="toolTip" >
         <string>Regular expression</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="textLabel2" >
        <property name="text" >
         <string>with</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KLineEdit" name="replaceEdit" >
        <property name="toolTip" >
         <string>Character string</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
   <zorder>spaceCheck</zorder>
   <zorder>asciiCheck</zorder>
   <zorder>vfatCheck</zorder>
   <zorder></zorder>
  </widget>
  <widget class="QGroupBox" name="groupingGroup" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>100</y>
     <width>486</width>
     <height>151</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize" >
    <size>
     <width>0</width>
     <height>150</height>
    </size>
   </property>
   <property name="title" >
    <string>File Naming Scheme</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" >
    <item>
     <widget class="QCheckBox" name="customschemeCheck" >
      <property name="toolTip" >
       <string>If checked, use a custom format string for naming the files in the collection</string>
      </property>
      <property name="text" >
       <string>Custo&amp;m Format</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="filetypeCheck" >
      <property name="toolTip" >
       <string>If checked, create a directory hierarchy using the filename extension.</string>
      </property>
      <property name="text" >
       <string>Group b&amp;y File Type</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="initialCheck" >
      <property name="toolTip" >
       <string>If checked, introduce another directory hierarchy for the artists' initials.</string>
      </property>
      <property name="text" >
       <string>Group &amp;by Artist's Initial</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="formatLabel" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>F&amp;ilename Format:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>formatEdit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KLineEdit" name="formatEdit" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="formatHelp" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>(Help)</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
   <zorder>customschemeCheck</zorder>
   <zorder>filetypeCheck</zorder>
   <zorder>initialCheck</zorder>
   <zorder></zorder>
  </widget>
  <widget class="QGroupBox" name="previewBox" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>390</y>
     <width>486</width>
     <height>55</height>
    </rect>
   </property>
   <property name="title" >
    <string>Destination Preview</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3" >
    <item>
     <widget class="QLabel" name="previewText" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip" >
       <string>This is what the file names will look like after renaming.</string>
      </property>
      <property name="text" >
       <string/>
      </property>
      <property name="wordWrap" >
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>42</y>
     <width>486</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <item>
     <widget class="QCheckBox" name="coverCheck" >
      <property name="text" >
       <string>&amp;Use Cover Art for Folder Icons</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="ignoreTheCheck" >
      <property name="toolTip" >
       <string>If checked, postfix artists' names starting with 'The' with ', The'.</string>
      </property>
      <property name="text" >
       <string>I&amp;gnore 'The' in Artist Names</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>450</y>
     <width>486</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <item>
     <spacer name="spacer2_2" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType" >
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>347</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QCheckBox" name="overwriteCheck" >
      <property name="toolTip" >
       <string>If checked, overwrite files of the same name without asking.</string>
      </property>
      <property name="text" >
       <string>Overwrite &amp;Destination</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>2</y>
     <width>486</width>
     <height>36</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <property name="margin" >
     <number>5</number>
    </property>
    <item>
     <widget class="QLabel" name="folderLabel" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip" >
       <string>Base directory under which to put files</string>
      </property>
      <property name="text" >
       <string>C&amp;ollection Folder:</string>
      </property>
      <property name="wordWrap" >
       <bool>false</bool>
      </property>
      <property name="buddy" >
       <cstring>folderCombo</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="KComboBox" name="folderCombo" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip" >
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="0" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local" >kcombobox.h</include>
  <include location="local" >klineedit.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>customschemeCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>formatEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customschemeCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>formatLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customschemeCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>initialCheck</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customschemeCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>filetypeCheck</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
